client/my-sites/exporter/export-card/index.jsx

```diff
@@ -77,7 +80,8 @@ class ExportCard extends Component {
 						onClickExport={ this.props.exportSelectedItems }
 					/>
 				</FoldableCard>
-				{ this.props.isExporting && <Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
+				{ this.props.isExporting && ! isJetpackSite &&
+				<Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
```

Comment:
I'm a bit wary of adding `isJetpack` checks in the code where it seems (I could be wrong) the difference in behaviour is mainly whether the export is sync/async. For example, if we wanted to make Jetpack exports async in future, these `isJetpack` conditions could be confusing. Maybe `isSync` would be better?

I'm also not 100% sure on this, but I don't think `isExporting` should be true in a sync export (since the status is 'starting' until the first request completes) so this check might not even be necessary.